packages/view/backend/sirius-components-view-gantt-edit/src/main/java/org/eclipse/sirius/components/view/gantt/provider/DeleteTaskDependencyToolItemProvider.java

```diff
@@ -0,0 +1,108 @@
+/**
```

Comment:
Wrong copyright and date

Reply:
It is generated code. I wonder if we should not let as the generation did it.

Reply:
done